app-preview/src/main/java/com/mapbox/navigation/examples/preview/copilot/CopilotActivity.kt

```diff
@@ -0,0 +1,236 @@
+                .accessToken(getString(R.string.mapbox_access_token))
+                .copilotOptions(
+                    // Set shouldSendHistoryOnlyWithFeedback to true if you want to sent Copilot traces
+                    // only when an end user submits negative feedback
+                    CopilotOptions.Builder().shouldSendHistoryOnlyWithFeedback(false).build()
+                )
+                .build()
+        )
+    }
+    private lateinit var routeResponse: DirectionsResponse
+
+    @SuppressLint("MissingPermission")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = MapboxActivityCopilotBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        copilotViewModel = ViewModelProvider(this)[CopilotViewModel::class.java]
+        routeResponse =
+            DirectionsResponse.fromJson(readRawFileText(this, R.raw.multiple_routes))
+
+        binding.startStopNavigation.setOnClickListener {
+            when (navigationSessionState) {
+                is NavigationSessionState.Idle -> {
+                    mapboxNavigation.setNavigationRoutes(emptyList())
+                    mapboxNavigation.startTripSession()
+                }
+                is NavigationSessionState.FreeDrive -> {
+                    mapboxNavigation.stopTripSession()
+                }
+                is NavigationSessionState.ActiveGuidance -> {
+                    mapboxNavigation.setNavigationRoutes(emptyList())
+                }
+            }
+        }
+
+        binding.setRoutes.setOnClickListener {
+            mapboxNavigation.setNavigationRoutes(
+                routeResponse.routes().toNavigationRoutes(RouterOrigin.Onboard),
+            )
+        }
+
+        binding.feedbackPush.setOnClickListener {
+            // Call postUserFeedback every time user submits feedback
+            mapboxNavigation.postUserFeedback(
+                FeedbackEvent.POSITIONING_ISSUE,
+                "Test feedback",
+                FeedbackEvent.UI,
+                "encoded_screenshot",
+            )
+            Toast.makeText(this, "Feedback event pushed!", Toast.LENGTH_SHORT).show()
+        }
+
+        binding.searchPush.setOnClickListener {
+            // Push a SearchResultsEvent every time Search results response is retrieved
+            MapboxCopilot.push(
+                SearchResultsEvent(
+                    SearchResults("mapbox", "https://mapbox.com", null, null, "?query=test1", null)
+                )
+            )
+            // Push a SearchResultUsedEvent every time a Search result is selected
+            MapboxCopilot.push(
+                SearchResultUsedEvent(
+                    SearchResultUsed(
+                        "mapbox",
+                        "test_id",
+                        "mapbox_poi",
+                        "mapbox_address",
+                        HistoryPoint(0.0, 0.0),
```

Comment:
nit: push a real location? seems like null would be supported if 0.0,0.0 was a real option